### DISCUSSION TITLE
(duplicate) Scrolling the mouse wheel to adjust game volume is not possible in the editor

### DISCUSSION THREAD

**Discussion by Haspamelodica** [ppy/osu#20745] [Ideas] [answered]
Edit: This is a duplicate of #10958 and #10960. Should have searched more before posting...

~~In all screens in lazer except for the editor, there is some region where scrolling the mouse wheel brings up the volume overlay. In the editor, every region has some other reaction to scrolling (scrubbing through the timeline, scrolling through the Setup screen, scrolling through beatmap problems in the Verify screen and so on). This means that you have to use the Settings menu (Ctrl+O) to adjust volume from the editor, which is very slow and unpractical.~~

~~Scrolling over the "menu bar" (the File, Edit, and View buttons) moves them left and right, which doesn't seem useful as there is more than enough room for those on sensible screen resolutions. Maybe scrolling there could adjust game volume instead?~~

**Comment by Haspamelodica** [comment]
Screenrecord showing the reactions of various editor screens to scrolling

https://user-images.githubusercontent.com/46954095/195612720-fc9fd03c-7d02-47b4-901d-b3d2e09a44fe.mp4

**Comment by IkaWaAyuMu** [comment, marked as answer]
OK if you still need a way to adjust volume inside editor, there is still a way. By enable the tool bar from `Ctrl+T` (or `Toggle Toolbar` shortcut) and use scroll wheel on `now playing` icon (the music notes icon). 
![image](https://user-images.githubusercontent.com/7066150/195614473-84605e2c-f3b1-4c1a-aeeb-5f3fd449733d.png)

and Alt+Scroll work. I forgot this exist :rofl:

**Comment by Haspamelodica** [comment]
From reading the pull request I also found out that pressing any modifier except Shift while scrolling also adjusts volume in most screens.
